### PR TITLE
update the UpgradeFail defination

### PR DESCRIPTION
**Pull request by zhiweiyin318**
related issue: https://github.com/open-cluster-management/backlog/issues/8151

### CODE REVIEW

**Review by hanqiuzh — APPROVED** [review]
looks good

**Comment by hanqiuzh** [comment]
it automatically added /lgtm after i approved through UI 🤯 
@showeimer  ^FYI

**Review by showeimer — APPROVED** [review]

**Pull request merged**
